ui/raidboss/data/05-shb/alliance/the_puppets_bunker.js

```diff
@@ -7,4 +7,288 @@
   timelineFile: 'the_puppets_bunker.txt',
   triggers: [
   ],
+  timelineReplace: [
+    {
+      'locale': 'de',
+      'replaceSync': {
+        '724P-operated superior flight unit \\(A-lpha\\)': '724P: Flugeinheit A-lpha',
+        '767P-Operated Superior Flight Unit \\(B-Eta\\)': '767P: Flugeinheit[p] B-eta',
```

Comment:
Did you mean to leave this as [p]?

Reply:
Good spot x.x Hat to fix them manually and forgot to remove it x.x